test/common/fit_common.py

```diff
@@ -267,7 +266,14 @@ def mkargs(in_args=None):
         in_args = sys.argv[1:]
 
     # command line argument parser returns cmd_args dict
-    arg_parser = argparse.ArgumentParser(description="Command Help")
+    arg_parser = argparse.ArgumentParser(
+        description="Command Help", add_help=False,
+        epilog="Note: -v/--verbosity is being deprecated. RAC-3984 is covering that work. " +
+               "See README.md for hints on the temporary mapping of -v to the new abilities " +
+               "as well as a few hints on using them directly. " +
+               "NOTE: below is the --help output from nosetests.")
```

Comment:
We discussed in the review to remove the output from nosetests when we do "run_tests.py -h", please.